src/ansible_navigator/utils/serialize.py

```diff
@@ -2,11 +2,27 @@
 
 import json
 
+from dataclasses import asdict
+from dataclasses import is_dataclass
 from typing import IO
 from typing import Any
 from typing import NamedTuple
 
 
+class EnhancedJSONEncoder(json.JSONEncoder):
```

Comment:
Looks like this could be in a dedicated PR.

Reply:
will be: #1014